packaging/DISTRIBUTIONS.md

```diff
@@ -0,0 +1,38 @@
+# Netdata distribution support matrix
+![](https://raw.githubusercontent.com/netdata/netdata/master/web/gui/images/packaging-beta-tag.svg?sanitize=true)
+
+In the following table we provide the list of officially supported operating systems.
+We detail the Distributions, flavours and the level of support Netdata is currently capable to provide.
+
+Kindly note that the following table is a work in progress. We have concluded on the list of distributions
+that we currently supporting and we are working on documenting our current state so that our users
+have complete visibility over the range of support
+
+Distribution | Family| Code health | Installer support | Kickstart support | Binary packaging support | Integrity testing (CI) | Functionality testing (CI) | Community support
+:------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :--------------------
+14.04.6 LTS (Trusty Tahr) | Ubuntu |  |  |  |  |  |  | 
+16.04.6 LTS (Xenial Xerus) | Ubuntu |  |  |  |  |  |  | 
+18.04.2 LTS (Bionic Beaver) | Ubuntu |  |  |  |  |  |  | 
+19.04 (Disco Dingo) Latest | Ubuntu |  |  |  |  |  |  | 
+Debian 7 (Wheezy) | Debian |  |  |  |  |  |  | 
+Debian 8 (Jessie) | Debian |  |  |  |  |  |  | 
+Debian 9 (Stretch) | Debian |  |  |  |  |  |  | 
+Debian 10 (Buster) | Debian |  |  |  |  |  |  | 
+Versions 6.* | RHEL |  |  |  |  |  |  | 
+Versions 7.* | RHEL |  |  |  |  |  |  | 
+Versions 8.* | RHEL |  |  |  |  |  |  | 
+Fedora 28 | Fedora |  |  |  |  |  |  | 
+Fedora 29 | Fedora |  |  |  |  |  |  | 
+Fedora 30 | Fedora |  |  |  |  |  |  | 
+Fedora 31 | Fedora |  |  |  |  |  |  | 
+CentOS 6.* | Cent OS | |  |  |  |  |  |  | 
+CentOS 7.* | Cent OS | |  |  |  |  |  |  | 
+CentOS 8.* | Cent OS | |  |  |  |  |  |  | 
+Open SuSE Leap 15.0 | Open SuSE |  |  |  |  |  |  | 
+Open SuSE Leap 15.1 | Open SuSE |  |  |  |  |  |  | 
+Open SuSE Tumbleweed (latest) | Open SuSE |  |  |  |  |  |  | 
+SuSE Enterprise Linux Server 11 | SLES |  |  |  |  |  |  | 
+SuSE Enterprise Linux Server 12 | SLES |  |  |  |  |  |  | 
+SuSE Enterprise Linux Server 15 | SLES |  |  |  |  |  |  | 
+Arch Linux (latest) | Arch |  |  |  |  |  |  | 
+All other linux | Other | |  |  |  |  |  | 
```

Comment:
Do we need `all other linux`? Sounds to vague. How can you define `Code health` or `Functionality testing (CI)` for `all other linux`? Is debian6 in all other linux? :smile:

Reply:
We have the static binary, that is for all other linux. Which reminds me i need to add a column about architecture

Reply:
I think we can remove the "all other linux" of this table, and we can write a paragraph saying that people can compile netdata on "All linux" case they have available the libraries.

Reply:
The various IoT Devices we want to support also probably fall on "all other linux", until we fine grain and go ahead and do more targeted support over those too

Reply:
> In the following table we provide the list of officially supported operating systems.

So we are officially supporting all distributions? Because specific + other = all.

I tend to agree with @thiagoftsm, maybe it would be better to move `other` outside the table. Not sure tho.

Reply:
"All other linux" implies the static 64 binary for any other linux distribution. I want to depict for that case what things we cover exactly.

Don't forget this is just a support matrix, so that we can quickly tell what we do for each distro.
Installation instructions explaining how to install netdata for all cases